core/src/main/java/org/apache/cloudstack/storage/command/TemplateOrVolumePostUploadCommand.java

```diff
@@ -49,7 +49,7 @@ public class TemplateOrVolumePostUploadCommand {
 
     String description;
 
-    private String defaultMaxAccountSecondaryStorage;
+    private String defaultMaxSecondaryStorage;
```

Comment:
```suggestion
    private String defaultMaxSecondaryStorageInGB;
```

update relevant methods as well, rest LGTM.

Reply:
changed the naming